specification/maps/data-plane/Route/stable/2025-01-01/route.json

```
@@ -0,0 +1,3783 @@
+          "type": "string"
+        },
+        "speedLimitInKilometersPerHour": {
+          "description": "Speed limit in kilometers per hour.",
+          "type": "number",
+          "format": "double"
+        }
+      }
+    },
+    "InputRouteRangeProperties": {
+      "type": "object",
+      "description": "Specifies the parameters to use for the calculation of isochrone polygon.",
+      "properties": {
+        "departAt": {
+          "description": "The date and time of departure from the origin point formatted as a dateTime value defined by [RFC 3339, section 5.6](https://www.rfc-editor.org/rfc/rfc3339#section-5.6). When a time zone offset is not specified, UTC will be assumed.\n\nIf the `departAt` is not set, the default value is the current time.\n\nExample:\n\n\"departAt\": \"2023-06-01T09:30:00.000-07:00\"",
+          "type": "string",
+          "format": "date-time"
+        },
+        "isSimplifiedPolygon": {
+          "description": "Use this to specify whether simplified polygons are required, reducing the number of vertices while preserving the overall shape. Currently, isSimplifiedPolygon only supports true. High-definition polygons are supported in version 2024-07-01-preview",
+          "type": "boolean",
+          "default": "true"
+        },
+        "optimizeRoute": {
+          "description": "Specifies the parameter to use to optimize the route. If not defined, the default is \"fastestWithoutTraffic\" which returns the route to minimize the travel time without using current traffic information.\n\nExample: \"optimizeRoute\":\"shortest\"",
+          "type": "string",
+          "default": "fastestWithoutTraffic",
+          "enum": [
+            "shortest",
+            "fastestWithoutTraffic",
+            "fastestWithTraffic"
+          ],
+          "x-ms-enum": {
+            "name": "RouteRangeOptimizeRouteEnum",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "shortest",
+                "name": "Shortest",
+                "description": "The route is calculated to minimize the distance. Traffic information is not used."
+              },
+              {
+                "value": "fastestWithoutTraffic",
+                "name": "FastestWithoutTraffic",
+                "description": "Finds the fastest route, without factoring in traffic information."
+              },
+              {
+                "value": "fastestWithTraffic",
+                "name": "FastestWithTraffic",
+                "description": "The route is calculated to minimize the time using current traffic information. `Note`: Only supported for driving and truck travelMode."
+              }
+            ]
+          }
+        },
+        "avoid": {
+          "$ref": "#/definitions/RouteRangeAvoidEnum"
+        },
+        "vehicleSpec": {
+          "$ref": "#/definitions/RouteRangeVehicleSpec"
+        },
+        "distanceBudgetInMeters ": {
```

This property name has a trailing space -- that's almost certainly wrong.
```suggestion
        "distanceBudgetInMeters": {
```